I18n fixes + polish

lucyhe opened the pull request:
- Rotate back arrows, add missing bdi tags, add missing translation
- Fix bug that always loaded English from storage
- Use XRegExp to detect if user-set or variable text (e.g. name, social network name) is right-to-left or left-to-right, and add unicode RTL and LTR characters around text to preserve its direction

<!-- Reviewable:start -->
[<img src="https://reviewable.io/review_button.png" height=40 alt="Review on Reviewable"/>](https://reviewable.io/reviews/uproxy/uproxy/1986)
<!-- Reviewable:end -->

jpevarnek commented:
Reviewed 9 of 9 files at r1.
Review status: all files reviewed at latest revision, 2 unresolved discussions, some commit checks failed.

---

<sup>**[src/generic_ui/scripts/translator.ts, line 17 \[r1\]](https://reviewable.io:443/reviews/uproxy/uproxy/1986#-K1AujSGAxwrrmq3Fa93)** ([raw file](https://github.com/uproxy/uproxy/blob/f80f48708fbe2fb9d69347fe6159102ae83d4376/src/generic_ui/scripts/translator.ts#L17)):</sup>
I think this might make more sense as a function (e.g.
```
function isRightToLeft(lang :string) :boolean {
  return XRegExp('[\\p{Arabic}\\p{Hebrew}]').test(lang);
}
```
)
<!-- Reviewable comment -K1AujSHJ5wRq5GtBC0q:-1378207217 -->

---

<sup>**[src/generic_ui/scripts/translator.ts, line 18 \[r1\]](https://reviewable.io:443/reviews/uproxy/uproxy/1986#-K1AuiMN1e2zeofxIXTx)** ([raw file](https://github.com/uproxy/uproxy/blob/f80f48708fbe2fb9d69347fe6159102ae83d4376/src/generic_ui/scripts/translator.ts#L18)):</sup>
Is this used anywhere?
<!-- Reviewable comment -K1AuiMN1e2zeofxIXTy:-318486094 -->

---


Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/uproxy/uproxy/1986)
<!-- Sent from Reviewable.io -->

lucyhe commented:
Also fixed tests, PTAL, thank you!

---

Review status: 2 of 11 files reviewed at latest revision, 2 unresolved discussions.

---

*[src/generic_ui/scripts/translator.ts, line 17 \[r1\]](https://reviewable.io:443/reviews/uproxy/uproxy/1986#-K1AujSGAxwrrmq3Fa93:-K4Nw3PiChkfBhE1ngAN:-499146009) ([raw file](https://github.com/uproxy/uproxy/blob/f80f48708fbe2fb9d69347fe6159102ae83d4376/src/generic_ui/scripts/translator.ts#L17)):*
Done.

---

*[src/generic_ui/scripts/translator.ts, line 18 \[r1\]](https://reviewable.io:443/reviews/uproxy/uproxy/1986#-K1AuiMN1e2zeofxIXTx:-K4Nw3oTpOwR4ss2Iqn8:-1054325829) ([raw file](https://github.com/uproxy/uproxy/blob/f80f48708fbe2fb9d69347fe6159102ae83d4376/src/generic_ui/scripts/translator.ts#L18)):*
removed

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/uproxy/uproxy/1986#-:-K4Nw5W7rDhPfTm0zYr1:277440624)*
<!-- Sent from Reviewable.io -->

jpevarnek commented:
:thumbsup:

---

Reviewed 9 of 9 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/uproxy/uproxy/1986#-:-K4NyfpQQ-EIjX1WI35n:1393940045)*
<!-- Sent from Reviewable.io -->
